dora/core/common/src/main/java/alluxio/worker/dora/OpenFileHandle.java

```diff
@@ -30,13 +31,17 @@ public class OpenFileHandle {
   private OutputStream   mOutStream; //outstream from UFS
```

Comment:
Can we rename this field to `mUfsOutStream `for more readable?

Reply:
OK